### PR TITLE
WhileScrolling Trigger

### PR DESCRIPTION
**Pull request by ichan-mb**
Trigger that will active while the ScrollView is in motion

**Example**:

```
<App>
	<ClientPanel>
		<Panel>
			<Rectangle ux:Name="indicator" Color="#838383" CornerRadius="5" Width="8" Height="80" Alignment="TopRight"
				Opacity="0" />
			<ScrollView ux:Name="scrollview">
				<ScrollingAnimation>
					<Move Y="0.95" Target="indicator" RelativeNode="scrollview" RelativeTo="Size" />
				</ScrollingAnimation>
				<WhileScrolling>
					<Change indicator.Opacity="1" Duration="0.1" DelayBack="1" />
				</WhileScrolling>
				<StackPanel ItemSpacing="10">
					<Each Count="100">
						<Panel Height="40" Color="#CDCDCD">
							<Text Value="Item #{index()}" Alignment="VerticalCenter" Margin="10" />
						</Panel>
					</Each>
				</StackPanel>
			</ScrollView>
		</Panel>
	</ClientPanel>
</App>
```

This PR contains:
- [ ] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Controls.ScrollView/Triggers/WhileScrolling.uno [hunk 1]
```diff
@@ -0,0 +1,60 @@
+using Uno;
+using Fuse;
+using Fuse.Triggers;
+using Fuse.Controls;
+
+/**
+	Trigger that will active while the @ScrollView is in motion
```
Comment (mortend) [inline Source/Fuse.Controls.ScrollView/Triggers/WhileScrolling.uno:7]:
```suggestion
	Trigger that will activate while the @ScrollView is in motion
```

**Review by mortend — COMMENTED** [review]
Just a small typo in the doc-comment. :)

File: Source/Fuse.Controls.ScrollView/Triggers/WhileScrolling.uno (hunk 1 above)
Comment (mortend) [reply Source/Fuse.Controls.ScrollView/Triggers/WhileScrolling.uno:7]: Or perhaps `... will become active ....`.

Comment (ichan-mb) [reply Source/Fuse.Controls.ScrollView/Triggers/WhileScrolling.uno]: fixed

**Review by mortend — APPROVED** [review]
Thank you! :)

**Comment by mortend** [comment]
- [x] https://travis-ci.org/github/fuse-open/fuselibs/builds/760922837

**Pull request merged**